#!/usr/bin/env python
PACKAGE = "lepton_thermal_sensor"

from dynamic_reconfigure.parameter_generator_catkin import *

gen = ParameterGenerator()

gen.add("temperature_threshold", double_t, 0, "Temperature Threshold", 50.0, 0, 300.0)

exit(gen.generate(PACKAGE, "lepton_thermal_sensor", "LeptonCamera"))


